src/potential/2b/energy2b.cpp

```diff
@@ -4,15 +4,25 @@ namespace e2b {
 
 double get_2b_energy(std::string m1, std::string m2, size_t nm,
                      std::vector<double> xyz1, std::vector<double> xyz2) {
+
+  // Order the two monomer names and corresponding xyz
   if (m2 < m1) {
     std::string tmp = m1;
     m1 = m2;
     m2 = tmp;
+    std::vector<double> tmp2 = xyz1;
```

Comment:
You might want to use a `std::move` here and below depending on your requirements.